bin/reacli.js

```diff
@@ -5,10 +5,39 @@ import program from "commander"
 import pkgInfo from "pkginfo"
 
 import { validateName, validatePath } from "../lib/utils/validators"
-import { createComponent } from "../lib/core"
+import { createComponent, createHook } from "../lib/core"
 
+const createElement = async ({ firstParam = null, pathsToComponentsToCreate = [], options = [] }) => {
+	// Cmd reacli component <path> creates a component architecture
+
+	const promises = []
+	for (let relativePath of pathsToComponentsToCreate) {
+		const path = pathModule.resolve(relativePath)
+
+		if (validatePath(path) && validateName(path)) {
+			try {
+				switch (firstParam) {
+				case "component":
+					promises.push(createComponent(path, options))
+					break;
+				case "hook":
+					promises.push(createHook(path, options))
+					break;
+				default:
+					program.outputHelp()
+					break;
+				}
+			} catch (error) {
+				console.log("ERROR: ", error)
+				program.outputHelp()
+			}
+		}
+	}
+
+	await Promise.all(promises)
```

Comment:
Hello, why do you need to use `Promise.all` here ? Only one promise is to be called... You could have awaited it write when calling it

Reply:
Heya, this is code from `develop` branch. See `https://github.com/reacli/cli/blob/develop/bin/reacli.js`. From what I understand, promises is an array - hence the `.all`

Reply:
Thank you. My bad !